sherpa/models/model.py

```diff
@@ -1118,10 +1118,14 @@ def __init__(self, name: str, parts: Sequence[Model]) -> None:
 
         Model.__init__(self, name, allpars)
 
+        # This check should probably be removed since it refers to
+        # loading a very-old session, and that is unlikely to work due
+        # to other changes in the code base.
```

Comment:
So, should it be removed? OK with me...

Reply:
I've created #2075 to track this, as it's separate from this PR